Kunena/Kunena-Forum
# Aurelia bootstrap optional

Pull Request for Issue:
When you have a bootstrap theme other then Joomla Build in, then Kunena will load (possibly) conflicting / double bootstrap files
 
#### Summary of Changes 
This change introduces load bootstrap template parameter (default = yes). When set to yes it will work as before, when set to no Kunena will not load the Joomla Bootstrap files but depends on the template to load the bootstrap files
 
#### Testing Instructions
On Cassiopedia Template:
before PR: load forum > all should be working
after PR: load forum > all should be working
